src/snyk/snykCode/views/suggestion/codeSuggestionWebviewProvider.ts

```diff
@@ -152,13 +161,24 @@ export class CodeSuggestionWebviewProvider
 
   private mapToModel(issue: Issue<CodeIssueData>): Suggestion {
     const parsedDetails = marked.parse(issue.additionalData.text) as string;
+    // get the workspace folders
+    // look at the filepath and identify the folder that contains the filepath
+    let workspaceFolder = '';
+    for (const element of this.workspace.getWorkspaceFolders()) {
+      if (issue.filePath.startsWith(element)) {
+        workspaceFolder = element;
+        break;
+      }
+    }
     return {
       id: issue.id,
       title: issue.title,
-      uri: issue.filePath,
+      uri: issue.filePath.replace(workspaceFolder + '/', ''),
```

Comment:
we should clean this up

we need this to make the filePath relative, which is needed for the ls

maybe might even make sense to move the logic out?